src/components/__test__/Home.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render, screen, act } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import store from '../../redux/store';
+import Home from '../Home/Home';
+
+test('renders the entire content without error', () => {
+  act(() => {
+    render(
+      <Provider store={store}>
+        <Home />
+      </Provider>,
+    );
+  });
+
+  act(() => {});
+
+  expect(screen.getByTestId('home-component')).toBeInTheDocument();
+});
```

Comment:
- [ ] Great job adding tests, but you need to also test
![image](https://github.com/rahimialijan/React-Capstone-Project/assets/62705947/89c68db0-56a1-4a43-b830-074e8b981b05)

Reply:
done